### DISCUSSION TITLE
How to remove Vercel Examples header and footer

### DISCUSSION THREAD

**Discussion by preferium** [vercel/examples#521] [Q&A] [answered]
I have just installed Vercel template and can't understand gow to remove Vercel Examples header and footer text. Can someone help me please?

<img width="1415" alt="Skjermbilde 2023-01-19 kl  10 44 15" src="https://user-images.githubusercontent.com/122758653/213409122-1a08d1ea-a158-4af5-8d14-d67ce351fe0a.png">

**Comment by dominiksipowicz** [comment, marked as answer]
Delete this line:
https://github.com/vercel/examples/blob/main/solutions/ai-chatgpt/pages/index.tsx#L25

**Reply by preferium** [reply]
Thank you! Works.

**Reply by granto** [reply]
@dominiksipowicz This worked great, Thanks!

**Comment by Fantastic-Beast** [comment]
> rcel template and can't understand gow to remove Vercel Examples header and footer text. Can someone help me please?

no longer working
